pytest.ini

```diff
@@ -1,3 +1,6 @@
 [pytest]
 addopts = --cov=morphapi
 
+filterwarnings =
+    ignore:distutils Version classes are deprecated:DeprecationWarning
+    ignore:The hookimpl CovPlugin.pytest_.* uses old-style configuration options:pytest.PytestDeprecationWarning
```

Comment:
I've opened an issue #31 to remind us to fix these at the root cause instead of ignoring them?
@adrien-berchet do you have any idea why these appear (if so, could you comment on #)?
From a quick google it seems some dependency (or us even) may be using distutils, which is deprecated.
Not made time to investigate the second warning yet.